phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -235,15 +235,22 @@ public static IndexVerifyType fromValue(byte[] value) {
     private static final Option END_TIME_OPTION = new Option("et", "endtime",
             true, "End time for indextool rebuild or verify");
 
+    private static final Option INCREMENTAL_VERIFY_OPTION = new Option("iv", "incre-verify",
```

Comment:
Nit: Please change this to full "incremental-verify"

Reply:
Should we call this as "retry-verify" since that's what the implementation looks like?